app/bundles/CampaignBundle/Tests/Controller/CampaignMapStatsControllerTest.php

```diff
@@ -64,30 +60,23 @@ private function getStats(): array
         ];
     }
 
-    /**
-     * @throws \ReflectionException
-     */
     public function testGetOptionLegendText(): void
```

Comment:
This method and the `testBuildMapData` only test the helper class, so it should be moved to the separate MapHelperTest class.

Reply:
right! I moved these tests to a separate file.